custom-words.txt

```diff
@@ -2844,4 +2844,5 @@ manytomany
 stransparent
 forceencryption
 tlsciphers
-tlsprotocols
\ No newline at end of file
+tlsprotocols
+ulimits
```

Comment:
Please create a separate PR to modify this file (custom-words.txt), as this is a common file, it's likely to bring merge conflict.